update_compilers/install_libraries.sh

```diff
@@ -288,3 +288,5 @@ get_or_sync_git_tags() {
 }
 
 get_or_sync_git_tags libs/ctre https://github.com/hanickadot/compile-time-regular-expressions.git master v2
+
+get_github_versions libs/vcl darealshinji/vectorclass v1.30
```

Comment:
Can you move this line to Line 178? It's a better place for it

Reply:
Sure! Done.